generators/server/templates/src/main/java/package/config/DatabaseConfiguration_cassandra.java.ejs

```diff
@@ -21,9 +21,6 @@ package <%= packageName %>.config;
 import com.datastax.oss.driver.api.core.data.TupleValue;
 import com.datastax.oss.driver.api.core.type.DataTypes;
 import com.datastax.oss.driver.api.core.type.TupleType;
-<%_ if (applicationTypeGateway && !databaseTypeCassandra) { _%>
```

Comment:
`!databaseTypeCassandra` condition inside a `_cassandra` file.

Reply:
Shouldn't you keep `<%_ if (applicationTypeGateway) {` ?